src/config/context.rs

```diff
@@ -70,5 +75,19 @@ pub const DISEASE_LABEL_OR_ID_CONTEXTS: [Context; 3] = [
     Context::OrphanetLabelOrId,
 ];
 
-pub const AGE_CONTEXTS: [Context; 3] =
-    [Context::SubjectAge, Context::OnsetAge, Context::AgeOfDeath];
+// NOT including DateOfBirth
```

Comment:
```suggestion
/// NOT including DateOfBirth
```

Reply:
I've noticed you're often leaking business logic into application logic. In other words, you're treating peripheral code - code that should just be infrastructure or plumbing - as if it's part of the core business domain.